hawc/apps/assessment/forms.py

```diff
@@ -1,6 +1,7 @@
 from pathlib import Path
```

Comment:
changes in this file seem appropriate. I'd recommend removing for this PR. For future work, we'll probably go app by app and replace the existing functionality and replace with the new